src/providers/workos.js

```diff
@@ -1,4 +1,6 @@
 export default function WorkOS(options) {
+  const apiUrl = options.apiUrl || 'api.workos.com';
```

Comment:
Could we call the new option `domain`? We already have it [reflected on the types](https://github.com/nextauthjs/next-auth/blob/main/types/providers.d.ts#L51) for other providers; once we figure out the way to express this via TS it's then easy for us to refactor.

Reply:
Done!